go/proof_data_test.go

```diff
@@ -13,18 +13,18 @@ type ExistenceProofTestStruct struct {
 	Expected []byte
 }
 
-func ExistenceProofTestData(t *testing.T) map[string]ExistenceProofTestStruct {
-	t.Helper()
+func ExistenceProofTestData(tb testing.TB) map[string]ExistenceProofTestStruct {
```

Comment:
Is it really necessary to change the interface if we're not doing benchmarking?

Reply:
We are doing fuzzing hence that change was needed and that isn't changing the interface as *testing.T implementing testing.TB so all existing code passes